### DISCUSSION TITLE
How does markmap decide the length of branches?

### DISCUSSION THREAD

**Discussion by james-tindal** [markmap/markmap#152] [Q&A] [answered]
How does markmap decide the length of branches based on the width of text? Where can I find this code in the repo?

**Comment by gera2ld** [comment, marked as answer]
https://github.com/markmap/markmap/blob/4d885ef42b1a16543553b427f1289dacf15332b6/packages/markmap-view/src/view.tsx#L257-L262

**Reply by james-tindal** [reply]
Thanks!
